research/gaia-x/pegasus/pegasus/src/api/concise/limit.rs

```diff
@@ -70,3 +71,10 @@ pub trait OrderLimit<D: Data + Ord> {
     /// [`limit()`]: crate::api::limit::Limit::limit()
     fn sort_limit(self, size: u32) -> Result<Stream<D>, BuildJobError>;
 }
+
+pub trait OrderLimitBy<D: Data> {
+    /// Sort and limit the input data stream via a user-defined comparator `cmp`.
+    fn sort_limit_by<F>(self, size: u32, cmp: F) -> Result<Stream<D>, BuildJobError>
```

Comment:
Can we simply call this function `order_limit_by()` ? Please help change all sort_xx() function to order_xx()

Reply:
fixed